tests/app/test_config.py

```diff
@@ -86,6 +87,24 @@ def test_queue_names_all_queues_correct():
     ) == set(queues)
 
 
+def test_no_celery_beat_tasks_scheduled_over_midnight_between_timezones(notify_api):
+    badly_scheduled_tasks = []
+
+    for task_name, task_info in notify_api.config["CELERY"]["beat_schedule"].items():
+        schedule = task_info["schedule"]
+        if not isinstance(schedule, crontab):
+            continue
+
+        if not (schedule.hour - {23}):
```

Comment:
took me a bit of local mucking around to realise that if a crontab is for something more frequent, it'll include every hour of the day in this. guess this makes sense, its hard to imagine a situation where we'd write, eg, crontab where it was {22, 23} or {11, 23} or something

Reply:
Yeah sorry I could include a comment there, let me add one.